test/plugin_functional/test_suites/saved_objects_management/hidden_types.ts

```diff
@@ -21,7 +21,6 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
   const esArchiver = getService('esArchiver');
   const testSubjects = getService('testSubjects');
 
-  // Failing: See https://github.com/elastic/kibana/issues/116059
   // Failing: See https://github.com/elastic/kibana/issues/116059
   describe.skip('saved objects management with hidden types', () => {
```

Comment:
This is still skipped. Is that intentional?

Reply:
Yes, I took a look at that one too but it wasn't related.
Couldn't help cleaning the duplicate comment though 😅